### ISSUE TITLE
X half diminished chords parsed as X flat 7 chords in iReal Pro part of ChoCo

### ISSUE THREAD

**Issue by rubencart** [smashub/choco#118]
```python
audiojams = jams.load('data/choco/v1.0.0/jams/ireal-pro_1132.jams')
audiojams.file_metadata.title
Out[39]: 'Serenity'
chordann = audiojams.annotations.search(namespace='chord_harte')[0]
chord_strings = [c.value for c in chordann.data if c.value != "N"]
chord_strings
Out[25]: 
['Db:7',
 'G:(3,5,b7,9,11,b13)',
 'Ab:(3,5,7,9,#11)',
 'Gb:(3,5,7,9,#11)',
 'E:min7',
 'A:7',
 'F:min7',
 'Bb:7',
 'Eb:maj7',
 'Db:maj',
 'G:7',
 'C:min7',
 'C:min7/b7',
 'Ab:min7',
 'Db:7',
 'G:7',
 'Gb:7',
 'F:7',
 'E:(3,5,7,9,#11)',
 'F:min7',
 'Bb:(3,#5,b7)',
 'Eb:maj7',
 'Db:7',
 'G:(3,5,b7,9,11,b13)']
```
Now if you check the serenity.jazz file included in the [iRealPro dataset](https://zenodo.org/records/3546040) you'll see that it starts with
```
1Dh7
=
1G7b13
=
2A-:maj7#11
2G-:maj7#11
```
The half diminished D chord is wrongfully parsed as a D flat dominant 7 chord. You can also find the lead sheet [here](https://www.irealb.com/forums/showthread.php?12753-Jazz-1410-Standards), starting with a half diminished chord. The [JHT dataset](https://github.com/DCMLab/JazzHarmonyTreebank) is based on the iRealPro dataset and it confirms this: the first chords of Serenity are `'D%7', 'G7', 'Ab^7', 'Gb^7', ...` with `D%7` representing D half diminished.

Same for Blues for Wood:
```python
meta.loc[meta.title == 'Blues For Wood']
Out[53]: 
                 id           title  ... movement_no movement_title
7123  ireal-pro_229  Blues For Wood  ...         NaN            NaN
[1 rows x 19 columns]

audiojams = jams.load('data/choco/v1.0.0/jams/ireal-pro_229.jams', strict=False, validate=False)
chordann = audiojams.annotations.search(namespace='chord_harte')[0]
chord_strings = [c.value for c in chordann.data if c.value != "N"]
chord_strings
Out[54]: 
['F:min7',
 'F:min7',
 'F:min7',
 'F:min7',
 'Db:7',
 'Db:7',
 'F:min7',
 'F:min7',
 'Db:7',
 'Bb:maj7',
 'Gb:7',
 'C:(3,5,b7,#9)',
 'F:min7']
```
The `'Gb:7'` chord should be G half diminished and not G flat as can be checked [here](https://www.irealb.com/forums/showthread.php?27435-1410-Jazz-Standards-Individual-Songs) or in the `bluesforwood.jazz` file in the iRealPro dataset. 

I don't know if this applies to only half-diminished chords or also to other chord types, to all half-diminished in iRealPro or even to all half-diminished chords in Choco.

**Comment by rubencart** [comment]
@andreamust @jonnybluesman Any update on this?

**Comment by andreamust** [comment]
Hi @rubencart,

Thank you for identifying this bug and providing a clear example of the issue. We’ve addressed the problem with the grammar rules used for the conversion of the partition in dbabf886c636a55a131d8bf00726bd8bee78db73. Additionally, in b2cd4b3cfc45567d7c27790a02103da994c83f59 we re-generated all the JAM files for the _ireal_pro_playlist_ partition (_ireal_pro_forum_ will be pushed soon, even though it is not currently officially supported in ChoCo).

Furthermore, we’ve enhanced the "prettify" function, resulting in more compact and readable chord labels.

All the changes will be included in the next release of ChoCo (v1.1.0). You can keep track of the changes that will be included in the release in this Issue: https://github.com/smashub/choco/issues/105.

Thanks again for your valuable feedback!

**Comment by rubencart** [comment]
Is it correct that we can already use the corrected jam files in the `partitions/.../jams-converted/` subdirectories on the main branch, before the next official release?
